app/data_client.go

```diff
@@ -109,6 +109,37 @@ type TabularData struct {
 	TimeReceived  time.Time
 }
 
+// ExportTabularDataReturn represents the result of an ExportTabularData API call.
+type ExportTabularDataReturn struct {
+	OrganizationID   string
+	LocationID       string
+	RobotID          string
+	RobotName        string
+	PartID           string
+	PartName         string
+	ComponentName    string
+	ComponentType    string
+	MethodName       string
+	TimeCaptured     time.Time
+	MethodParameters map[string]*GenericProtoMessage
```

Comment:
I'm not sure I understand this. Could you explain the type choice?

Reply:
But that's not what we use in TabularDataByFilter?

Reply:
I haven't seen the actual proto version so this is the reference I made from Vijay's code, but the return types for `TabularDataByFilter` and `ExportTabularData` are different.

Reply:
Ya, that's what I meant. `MethodParameters` inside `CaptureMetadata`

Reply:
@vijayvuyyuru Please hold onto this PR. I'll try to look into why we did it this way for `CaptureMetadata`. I didn't know that there was another case of `anypb.Any` being used here.

Reply:
Okay, @vijayvuyyuru sorry to give you more work, but I got some more background on this `Any` type, and realized that maybe going the same route that the `CaptureMetadata` did is better in terms of readability! I'll make those changes on my side as well. It shouldn't be too big of a fix for you, since `methodParamsFromProto` already exists in the data client. You should be able to directly call that method!